### PR TITLE
Created rails app with book model.

### PR DESCRIPTION
**Pull request by justinlittman**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -9,11 +12,67 @@ gem 'gpgme'
...
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+# Using Rails 6 so that have bulk insert.
+gem 'rails', github: 'rails/rails'
+# gem 'rails', '~> 5.2.3'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3'
+# Use Puma as the app server
+gem 'puma', '~> 3.11'
+# Use SCSS for stylesheets
+gem 'sass-rails', '~> 5.0'
```
Comment (jcoyne) [inline Gemfile]: I'd recommend generating your rails app with the `--api` flag.  We don't need sass/turbolinks/uglifier/coffee-rails for this project

Comment (justinlittman) [reply Gemfile]: Done.

File: config/credentials.yml.enc [hunk 2]
```diff
@@ -0,0 +1 @@
+95J/DUVNVRdOA7M8ibcz5S0FhsgQ+DcWP5z8JLvTnGNtFI0yzJjhPp/BHoWDWxG6ZvRpntpVuj8gitHilNP6X53XTlvpT+bV1m1/cd2h5HJiSdM/NSrKKwUyMGfTA3f+aG4FKZFjWfXtymwB662AaWHR7RfjiOTi3J02LQF2v2HefQxGKwQAw6rLOZemFUBEWZULQm8mD2BUUd/pNspm6Xu0xSANPe2oYwV7KaAoqUT4WIa0dtT9M9xkMUGI59tNKZkKfx40nzgre/go9OrAV1KoYN69TcOrutxTjCslnNv7Q2sixz1aUghy+iMD5rih62DGV5YMOCr0+8vHySyTzLf3xC0mindqKJAnyfO+6+oB31BooblJ5R6uzto3iUZcF2jt+KrMLZQ+LKt3oYuM3QTtpSMy5qgz2Md/--+HSR8v2UCR4w5Os1--GnicMF3131gxlG1nWBORJg==
```
Comment (aaron-collier) [inline config/credentials.yml.enc]: Do we want / need this in the repo? Or should it be in shared_configs?

File: app/services/fetcher_service.rb [hunk 3]
```diff
@@ -0,0 +1,139 @@
+# frozen_string_literal: true
```
Comment (aaron-collier) [inline app/services/fetcher_service.rb]: This is a great start. It feels to me like this service is doing too much and should be broken up into single responsibility services (i.e. download_service, status_service, etc...)

File: .gitignore [hunk 4]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (jcoyne) [inline .gitignore]: Cruft?

File: config/credentials.yml.enc (hunk 2 above)
Comment (jcoyne) [reply config/credentials.yml.enc]: The credentials.yml.enc should be in the repo.  The master.key needs to go into shared_configs.

File: .gitignore (hunk 4 above)
Comment (justinlittman) [reply .gitignore]: Actually, a mess.

**Comment by justinlittman** [comment]
@jcoyne Can you re-review at your convenience?

**Pull request merged**